modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -18,14 +18,13 @@ The Anypoint API Community Manager Search Box component enables users to search
 
```

Comment:
These are UI labels and follow the capitalization in the UI, and because they are the items in a definition list, they are already bold on the page: https://docs.mulesoft.com/api-community-manager/acm-lightning-components . However, I have created a Jira story to update this file with asterisks, to prevent ambiguity during reviews of the source file. https://www.mulesoft.org/jira/browse/DOCS-11178